### ISSUE TITLE
Che "run" command fails for Wildfly Swarm quickstarts

### ISSUE THREAD

**Issue by ldimaggi** [openshiftio/openshift.io#2103]
```
command: cd /projects/january29test && scl enable rh-maven33 'java -jar target/*-swarm.jar'
preview:http://wildfly-8a2cjv3a3nfps71x-ldimaggi-che.8a09.starter-us-east-2.openshiftapps.com
Error: Unable to access jarfile target/*-swarm.jar
```

![screenshot from 2018-01-29 15-37-28](https://user-images.githubusercontent.com/642621/35533186-b573d350-050a-11e8-9adc-2d10be20da04.png)

The run command is not doing a build before it tries to do a run.

**Comment by ibuziuk** [comment]
@ldimaggi have you build the project before executing run command e.g. did you have swarm jar before executing `java -jar`

**Comment by ldimaggi** [comment]
That works - but - shouldn't the run command also perform a build?

**Comment by ibuziuk** [comment]
nope, but I agree that this could be a bit misleading if one have not get used to the way Che IDE commands suppose to work in comparison with Descktop IDE. So, if you think it is important usability problem the issue upstream could be opened https://github.com/eclipse/che

**Comment by ibuziuk** [comment]
@ldimaggi actually giving another thought I think we could improve the situation with run comands [1]
 - mvn package && java -jar ./target/example-jpa-war-swarm.jar
 - mvn wildfly-swarm:run
The second option is more preferable, but might require quickstart update.

[1] https://github.com/wildfly-swarm/wildfly-swarm-examples/tree/master/jpa/jpa#run

**Comment by ibuziuk** [comment]
`wildfly-swarm` plugin is not installed in quickstarts, so in order to proceed we need to add this plugin to all wildfly quickstarts:
 
`[ERROR] No plugin found for prefix 'wildfly-swarm' in the current project and in the plugin groups [org.apache.maven.plugins, org.codehaus.mojo] available from the repositories [local (/home/user/.m2/repository), central (https://repo1.maven.org/maven2)] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/NoPluginFoundForPrefixException`
